### ISSUE TITLE
Build failure when CSS file contains backtick character or other special syntax

### ISSUE THREAD

**Issue by Haprog** [bennypowers/lit-css-loader#1]
We ran into this issue when trying to use `lit-css-loader` to load a file like https://github.com/vaadin/skeleton-starter-flow/blob/master/frontend/styles/vaadin-text-field-styles.css which causes the webpack build to fail because the CSS contains [a comment](https://github.com/vaadin/skeleton-starter-flow/blob/master/frontend/styles/vaadin-text-field-styles.css#L7) which includes a backtick (`` ` ``) character.

```css
/* Example: the style is applied only to the textfields which has the `bordered` theme attribute. */
:host([theme~="bordered"]) [part="input-field"] {
  ...
```

Webpack shows an error like "Module parse failed: Unexpected token (9:71)" when trying to execute the module generated by lit-css-loader.

When looking at the implementation of `lit-css-loader`, the bug is pretty obvious (after seeing this error).

Basically any syntax that has special meaning inside a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) would need to be escaped in the `source` parameter to be safe. To my understanding it should be enough to escape:

- the backtick: `` ` `` -> `` \` ``
- the expression syntax: `${` -> `\${`
- and backslash itself: `\` -> `\\`

In CSS these are most likely to occur in some comments that may contain example code or documentation in markdown syntax, but it's also possible for them to be in CSS string values e.g. `content: '${';`

Here's a minimal project that reproduces the issue: https://github.com/Haprog/repro-lit-css-loader-1

I already tested a simple fix for this. I will create a pull request with my fix.
